### PR TITLE
Add cryptography to run deppies

### PR DESCRIPTION
**Pull request by igortg**

### CODE REVIEW

**Comment by igortg** [comment]
@conda-forge-admin, please rerender

**Comment by igortg** [comment]
@conda-forge/core, can I get some help on rerender this one?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -72,4 +73,4 @@ extra:
     recipe-maintainers:
         - kwilcox
         - pmlandwehr
-        - sodre
+- sodre
```
Comment (scopatz) [inline recipe/meta.yaml]: It seems like this line change is causing issues with rerender. if you revert it, that should fix it

**Comment by igortg** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (igortg) [reply recipe/meta.yaml]: Aff... tks. Sorry to bother you with such dumb issue

**Comment by ocefpaf** [comment]
@conda-forge-admin, please rerender

**Pull request merged**